elasticreindexer/README.md

```diff
@@ -1,13 +1,89 @@
+To do that have to use the `indices-creator` tool:
+
+    ```
+    cd elasticreindexer
+    cd cmd/indices-creator/
+    go build 
+    ```
+  
+- After the build is done have to update the `config/cluster.toml` file with the information about the Elasticsearch cluster. In the `cluster.toml` have to set the URL 
+of the Elasticsearch cluster, and for what indices have to create the mappings.
+
+- Optionally also the mappings can be customized ( open file with mappings for every index and set different settings, based on the needs)
+
+- Run `./indices-creator` in order to create all the indices and mappings
+
+***
+
+_*This step can be skipped for the clusters the already have information indexed_ 
+
+***
+
+### STEP 2
+- After the mappings and indices was created we can start to copy all the information. In order to do this have to use the `elasticreindexer` tool that 
+will copy every index one by one.
+
+    ```
+    cd elasticreindexer
+    cd cmd/elasticreindexer/
+    go build 
+    ```
+- Update the Elasticsearch instance configuration for both source and destination inside `config.toml`. In the `config.toml` file have to set the 
+URL of the `input` Elasticsearch instance ( the one from where have to copy all the information) and the `output` instance ( the one where all the information 
+from the `input` instance will be copied )
+- The `config.toml` file contains by default all the Elasticsearch indices that are populated by an Elrond observing-squad
+
+- Also, if you want to copy indices with timestamp have to set in the `config.toml` file the `blockchain-start-time` ( by default is the one from the mainnet)
+
+- Run `./elasticreindexer --skip-mappings` ( will start to reindex all the information from the input cluster in the output cluster based on the `config.toml` file)
+
+***
+
+_** Don't start the observing-squad only after the indices `accounts`, `accountsesdt` and `tokens` are copied_
+
+***
+
+#### SPEED UP STEP 2
+- The `STEP 2` will take a lot of time because are `hundreds of gigabytes` of data and in order to speed up the process we can do the 
+next things:
+
+1. Run an instance of `elasticreindexer` only with the indices from the list `indices-no-timestamp`
+2. Run multiple instance of `elasticreindexer` with the next configurations for `indices-with-timestamp`:
+
+    `a.` `indices-with-timestamp = ["accountsesdt", "tokens"]`
+
+    `b.` `indices-with-timestamp = ["blocks", "receipts", "rounds"`
+
+    `c.` `indices-with-timestamp = ["transactions", "miniblocks", "scdeploys"]`
+
+    `d.` `indices-with-timestamp = ["accountshistory", "scresults"]`
+
+    `e.` `indices-with-timestamp = ["accountsesdthistory"]`
+
+    `f.` `indices-with-timestamp = ["logs"]`
+
+    `g.` `indices-with-timestamp = ["operations"]`
+
+*** 
+
+_*** For all the configs from `a-g` the field `indices-no-timestamp` have to be empty or commented_
```

Comment:
Maybe we can add a flag into the binary --with-timestamp. So you can have both kinds of configs.

Reply:
We can have both types of configs but if both are populated it will copy everything.